<launch>
	<!-- To push joint velocities:
	     rostopic pub -r 10 /cmd_vel sensor_msgs/JointState '{velocity: [0, 0,, 0, 0, 0]}'
        -->
	<!-- Robot -->
	<node pkg="visp_ros" type="visp_ros_viper850_node" name="viper850" output="screen">
		<!-- end_effector = ["TOOL_PTGREY_FLEA2_CAMERA" (default), "TOOL_GENERIC_CAMERA", "TOOL_CUSTOM"] 
		if CUSTOM_TOOL use custom_tool_transformation_filename parameter to set the location of eMt transformation
                 -->
		<param name="end_effector" value="TOOL_PTGREY_FLEA2_CAMERA" />
		<!-- set_control_mode = ["joint_space", "camera_frame (default)", "reference_frame"] -->
		<param name="set_control_mode" value="joint_space" />
		<param name="cmd_vel_topic_name" value="/cmd_vel" />
		<!-- get_state_space = ["joint_space", "camera_frame (default)", "reference_frame"] -->
		<param name="get_state_space" value="joint_space" />
	</node>
</launch>

